### PR TITLE
[FluidDynamicsApplication] Base elements boundary integrals and tests upgrade

### PR DESCRIPTION
**Pull request by rubenzorrilla**
@jcotela as we discussed, I've renamed the boundary integrals method to  AddBoundaryTraction (former AddBoundaryIntegral) and corrected the LHS and RHS signs. I also took the chance to separate some of the cpp tests in several files (the previous one became extremely large).

### CODE REVIEW

File: applications/FluidDynamicsApplication/custom_elements/fluid_element.h [hunk 1]
```diff
@@ -433,7 +433,7 @@ class FluidElement : public Element
         TElementData& rData,
         MatrixType& rMassMatrix);
 
-    virtual void AddBoundaryIntegral(
+    virtual void AddBoundaryTraction(
```
Comment (jcotela) [inline applications/FluidDynamicsApplication/custom_elements/fluid_element.h:449]: Could you add a doxygen comment explaining what this function should do in derived implementations?

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_embedded_fluid_element.cpp [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+namespace Testing {
+
+KRATOS_TEST_CASE_IN_SUITE(EmbeddedElement2D3N, FluidDynamicsApplicationFastSuite)
+{
+    ModelPart model_part("Main");
+    model_part.SetBufferSize(3);
+
+    // Variables addition
+    model_part.AddNodalSolutionStepVariable(DENSITY); // TODO: To be removed once the element migration is finally finished
+    model_part.AddNodalSolutionStepVariable(DYNAMIC_VISCOSITY); // TODO: To be removed once the element migration is finally finished
```
Comment (jcotela) [inline applications/FluidDynamicsApplication/tests/cpp_tests/test_embedded_fluid_element.cpp]: Can we remove these two in the test? Do the navier_stokes elements use a constitutive law?

**Review by jcotela — DISMISSED** [review]

File: applications/FluidDynamicsApplication/custom_elements/fluid_element.h (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/custom_elements/fluid_element.h:449]: Sure! Actually, I completely forgot to do it...

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_embedded_fluid_element.cpp (hunk 2 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/tests/cpp_tests/test_embedded_fluid_element.cpp]: I will try. Yes, the navier_stokes elements use a constitutive law.

File: applications/FluidDynamicsApplication/custom_elements/fluid_element.h (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/custom_elements/fluid_element.h:449]: Done, if it isn't clear enough tell me something.

**Review by jcotela — APPROVED** [review]
Thanks @rubenzorrilla. I've modified the comment slightly to make it clearer that this term is needed in embedded implementations. If you are ok with the changes, just merge the PR.

**Pull request merged**